pvsystemprofiler/utilities/hour_angle_equation.py

```diff
@@ -6,18 +6,19 @@
 from pvsystemprofiler.utilities.time_convert import clock_to_solar
 
 
-def calculate_omega(data_sampling, num_days, lon, doy, gmt_offset):
+def calculate_omega(doy, data_sampling, lon, gmt_offset):
```

Comment:
Looks like this is still called once in [tests/fixtures/tilt_azimuth/tilt_azimuth_estimation_data_creator.ipynb](https://github.com/slacgismo/solar-data-tools/blob/0a8c2db4d12f37fd684abe19f247f9e36b60e0a8/tests/fixtures/tilt_azimuth/tilt_azimuth_estimation_data_creator.ipynb) using the previous order of params. Should we update it or was that notebook just used once for creating data for tests?

Reply:
I should take a look at that notebook to make sure. thanks for finding that.